src/Resources/Resources.Test/UnitTests/Utilities/BicepUtilityTests.cs

```diff
@@ -0,0 +1,131 @@
+﻿using System.Collections.Generic;
```

Comment:
Please add the Microsoft licence header for newly created files, below is an example:
https://github.com/Azure/azure-powershell/blob/8db14b5c736b9c4d24578391e0512c28f651e25c/src/Resources/Resources.Test/ScenarioTests/ManagedApplicationTests.cs#L1-L13